docs/conf.py

```diff
@@ -25,9 +25,9 @@
 author = 'Ksenia Zaytseva, David Lougheed, Simon Chenard'
 
 # The full version, including alpha/beta/rc tags
-release = '0.3.0'
-
+release = '0.4.0'
 
+version = '0.4.0'
```

Comment:
If this is the project version, we should make a release checklist or find a way to sync this with setup.py

Reply:
I can remove it for now till we have a decision